### PR TITLE
Test in Laravel 8

### PR DESCRIPTION
**Pull request by stefro**

### CODE REVIEW

**Comment by stefro** [comment]
I'm sorry for the 2 PR's I forgot to add Laravel 8 to Travis.

File: composer.json [hunk 1]
```diff
@@ -20,11 +20,11 @@
...
         "guzzlehttp/guzzle": "^6.3.1|^7.0",
-        "illuminate/contracts": "^7.25",
-        "illuminate/support": "^7.25"
+        "illuminate/contracts": "^7.25|^8.0",
+        "illuminate/support": "^7.25|^8.0"
     },
     "require-dev": {
-        "laravel/framework": "^7.25",
+        "laravel/framework": "^7.25|^8.0",
         "orchestra/testbench": "^5.0",
```
Comment (BenSampo) [inline composer.json:28]: Testbench will need to be updated too:

`"orchestra/testbench": "^5.0|^6.0",`

**Comment by BenSampo** [comment]
Thanks 👍

**Comment by stefro** [comment]
Thank you for the great package!

**Pull request merged**